packages/govuk-frontend/src/govuk/govuk-frontend-component.mjs

```diff
@@ -29,4 +29,35 @@ export class GOVUKFrontendComponent {
       throw new SupportError()
     }
   }
+
+  /**
+   * @protected
+   * @param {Schema} schema
+   * @param {Config[ConfigKey]} config
+   * @returns {boolean} Whether the config is valid
+   */
+  validateConfig(schema, config) {
```

Comment:
@domoscargin Do we need to follow `checkSupport()` here?

For example:

1. Add an `isConfigValid()` utility to return true/false
2. Use method `this.checkConfig()` to throw an error